integration_tests/configs/default.jsonnet

```diff
@@ -30,6 +30,7 @@
         memiavl: {
           enable: true,
           'zero-copy': true,
+          'snapshot-interval': 5,
```

Comment:
seems need enlarge default a bit to allow [statesync](https://github.com/crypto-org-chain/cronos/blob/b90ff52ac2fac6936e39119967e822b3669cadf1/integration_tests/test_basic.py#L264) test to continue

Reply:
trying locally, it should has no impact on the test?

Reply:
there's a deadlock.

Reply:
yea since [commit](https://github.com/crypto-org-chain/cronos/blob/dcfc34bbdb1fe18956ea98011fba126e0c9f5ff0/memiavl/db.go#L260-L261) already lock

Reply:
see some wrong Block.Header.AppHash error